src/components/choice-tabs/ChoiceTabs.scss

```
@@ -0,0 +1,268 @@
+// ---------------------------------------------------------------------------
+.choice-tabs__card {
+  position: relative;
+  z-index: 0;
+  flex: 1;
+  min-width: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 0;
+  padding: 18px 20px;
+  border-radius: 14px;
+  // Same adaptive tint as shared-tabs so the cards stay visible in light
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)